docs/examples/helpbutton/withLink.js

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+import { type Node } from 'react';
+import { HelpButton, Text, Flex } from 'gestalt';
+
+export default function Example(): Node {
+  const onHandleClick = () => {
```

Comment:
This isn't necessary. We don't use the console in our docs examples.

Reply:
Learned++